packages/core/src/Mode.ts

```diff
@@ -15,7 +15,10 @@ export interface ModeDefinition {
 /**
  * Properties of a `VNode` that can be modified by a mode.
  */
-export type ModeProperty = 'editable' | 'breakable';
+export enum ModeProperty {
+    EDITABLE = 'editable',
+    BREAKABLE = 'breakable',
+}
```

Comment:
Ah bah il l'a fait. Si j'ai bien suivi l'autre conversation, nby disapproves. Ou alors il s'en fout tant que ça vient pas dans la définition même de `Rule` ? (Ouais parce que c'est `RuleProperty` en fait non ?) Comme dit dans l'autre thread de commentaire, j'ai pas trop compris ce qu'il voulait.

Reply:
Oui je désaprouve a 100%. `is` est vraiment plus difficile a grepper. En plus on a deja le meme nom de methode sur `AbstractNode`.

En quoi c'est mieux d'avoir `is(ModeProperty.EDITABLE)` que de faire `isEditable`.

On y gagne pas grand chose non plus dans la definition car il faut une ligne par property.
Avec les méthodes, il faut une methode de une ligne (qu'on peut folder) par property.